front/app/containers/ProjectFolderShowPage/ProjectFolderDescription.tsx

```diff
@@ -216,7 +218,7 @@ const ProjectFolderDescription = memo<Props & WrappedComponentProps>(
               )}
           </Description>
           {!isNilOrError(projectFolderFiles) &&
```

Comment:
Can we remove this isNilOrError check. All we need is to check that projectFolderFiles is there.